### ISSUE TITLE
Screenreader does not read the name of the "Automatic" color that are selected for "Text Color" and "Background Color" controls of CK Editor.

### ISSUE THREAD

**Issue by sagar17sep** [ckeditor/ckeditor4#1946]
BUG-

## Provide detailed reproduction steps (if any)

1. Bring Focus on CK Editor toolbar and Navigate using Alt + F10  to "Color Toolbar" section.
2. Start Windows Narrator.
3. Expand the flyouts for "Text Color" and "Background Color" controls to read the list items.
4. Navigate to "Automatic" list item in the lists.

### Expected result

Windows Narrator should read out the color of the default color(Automatic) selected.
For "Text Color" : Automatic color- Black
For  "Background Color" : Automatic color- White

### Actual result

The Windows Narrator does not read which color is automatically selected as default color for "Text Color" and "Background Color" controls. Only by visualization it is possible to know that "Black" and "White" are the default colors. Narrator gives no information regarding the same.

## Other details

* Browser: … Egde 41.16299.248.0 & IE 11.309.16299.0
* OS: …Windows 10
* CKEditor version: …4.6.2
* Installed CKEditor plugins: …

**Comment by mlewand** [comment]
This is a feature request rather than bug report. It would require a fair amount of code to generate labels from any color. Add to that the fact that colors should be localized.

This is a valid feature request, but requires some substantial efforts to get it done so I don't think we'll be able to work on it anytime soon.

In case there are contributors willing to make a PR, this feature should be contained in a separate plugin as it will affect the editor size.

In case you are licensed customer you might want to [contact CKSource](https://ckeditor.com/contact/) to check what possibilities for creating such feature extended support can offer.

**Comment by Comandeer** [comment]
It's true that supporting all colours will be problematic. However we could introduce limited support for colour labels, only for colours that are present in our colours listbox. As we already have their names, adding them as a part of automatic colour label should be an easier task.
